runtime/browser/runtime.h

```diff
@@ -42,8 +47,12 @@ class Runtime {
   virtual void OnLowMemory();
 
  private:
-  WebApplication* application_;
-  NativeWindow* native_window_;
+  void InitWebApplication(NativeWindow::Type window_type);
+
+  std::unique_ptr<WebApplication> application_;
+  std::unique_ptr<common::ApplicationData> application_data_;
+  std::unique_ptr<NativeWindow> native_window_;
+  Ewk_Context* ewk_context_;
```

Comment:
ewk context should own by WebApplication
Please try to fix the crash issue, 
If crash was happen by Chromium-efl code, please report to WebEngine team crash callstack